### PR TITLE
Coin listing structure initial draft docs

### PR DESCRIPTION
**Pull request by nateshmbhat**
Issue : #49

### CODE REVIEW

**Review by hussu010 — COMMENTED** [review]
```
priceJsonPath: {
    type: String,
    required: true,
  },
  ///See https://www.npmjs.com/package/jsonpath for syntax reference
  volumeJsonPath: {
    type: String,
    required: true,
  },
```
are these the endpoints to get the price and volume of a coin? if yes, we can change it to `tickerJsonPath` since both price and the volume will be returned by the same endpoint.

**Comment by nateshmbhat** [comment]
@dhakal0kushal these are not end points. These are json path in the response at which price and volume data can be found.

**Comment by hussu010** [comment]
> @dhakal0kushal these are not end points. These are json path in the response at which price and volume data can be found.

are not we storing the endpoints to get price and volume somewhere?

File: docs/coinListing/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    type: mongoose.Types.ObjectId,
+    ref: 'Exchange',
+    required: true,
+  },
+  coin: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+    ref: 'Coin',
+  },
+  endPointPath: {
```
Comment (hussu010) [inline docs/coinListing/README.md:29]: i guess this is the endpoint path then?

**Comment by nateshmbhat** [comment]
The end point will be part of exchange schema itself. We shall discuss this in more detail in our meeting on Sunday.

I will explain everything there

**Review by hussu010 — COMMENTED** [review]
looks good to me
will need to discuss more in tomorrow's meeting

File: docs/coinListing/README.md (hunk 1 above)
Comment (nateshmbhat) [reply docs/coinListing/README.md:29]: This is only the path parameters for the actual end point. Actual end point will be part of exchange schema

**Pull request merged**